application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -85,13 +85,13 @@
             "852509694341283871", DOODLECREW_NAME, "878067389634314250", WORDSNACK_NAME,
             "879863976006127627", LETTERTILE_NAME, "879863686565621790");
 
-    private static final List<OptionData> inviteOptions = List.of(
-            new OptionData(OptionType.INTEGER, MAX_USES_OPTION,
-                    "Max uses for invites, 0 for infinite, 100 is the top limit.", false)
-                        .setRequiredRange(0, MAX_USES_LIMIT),
+    private static final List<OptionData> inviteOptions = List.of(new OptionData(OptionType.INTEGER,
+            MAX_USES_OPTION,
+            "How many times this invite can be used, 0 infinite (default) - 100 being the highest.",
+            false).setRequiredRange(0, MAX_USES_LIMIT),
             new OptionData(OptionType.INTEGER, MAX_AGE_OPTION,
-                    "Max age in seconds. Set this to 0 to never expire, default is 1 day", false)
-                        .setRequiredRange(0, MAX_AGE_LIMIT));
+                    "How long, in days this activity can be used before it expires, 0 (No expiry), Maximum is 7 days. ",
+                    false).setRequiredRange(0, MAX_AGE_LIMIT));
```

Comment:
The variable name should include the unit to prevent bugs `MAX_AGE_DAYS_LIMIT`

Reply:
Changed variable name.